StripeFinancialConnections/StripeFinancialConnections/Source/Native/LinkAccountPicker/LinkAccountPickerBodyView.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  LinkAccountPickerBodyView.swift
+//  StripeFinancialConnections
+//
+//  Created by Krisjanis Gaidis on 2/13/23.
+//
+
+import Foundation
+@_spi(STP) import StripeCore
+@_spi(STP) import StripeUICore
+import UIKit
+
+@available(iOSApplicationExtension, unavailable)
+protocol LinkAccountPickerBodyViewDelegate: AnyObject {
+    func linkAccountPickerBodyView(
+        _ view: LinkAccountPickerBodyView,
+        didSelectAccount selectedAccount: FinancialConnectionsPartnerAccount
+    )
+    func linkAccountPickerBodyViewSelectedNewBankAccount(
+        _ view: LinkAccountPickerBodyView
+    )
+}
+
+@available(iOSApplicationExtension, unavailable)
+final class LinkAccountPickerBodyView: UIView {
+
+    private let accounts: [FinancialConnectionsPartnerAccount]
+    weak var delegate: LinkAccountPickerBodyViewDelegate?
+
+    private lazy var verticalStackView: UIStackView = {
+        let verticalStackView = UIStackView()
+        verticalStackView.axis = .vertical
+        verticalStackView.spacing = 12
+        return verticalStackView
+    }()
+
+    init(accounts: [FinancialConnectionsPartnerAccount]) {
+        self.accounts = accounts
+        super.init(frame: .zero)
+        addAndPinSubview(verticalStackView)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func selectAccount(_ selectedAccount: FinancialConnectionsPartnerAccount?) {
+        // clear all previous state
+        verticalStackView.arrangedSubviews.forEach { arrangedSubview in
+            arrangedSubview.removeFromSuperview()
+        }
```

Comment:
debatable :P but I am changing it to your suggestion! 

what I wrote is technically the 'pure' answer of typing everything out explicitly, the extra lines also "pop it out" more so its clear this is an important part of function

I think a 1-liner "tucks" this logic away, which makes it almost seem like it does not exist, or is some cookie-cutter code. For example, devs tend to do that with `guard let self = self else { return }` 

One could rephrase this debate as: "Is it worth emphasizing this logic?" (3-lines emphasize, 1-liner tucks it away) And I _feel_ like the emphasize wins the debate because this is quite a critical part of the function.

I'd be curious on your analysis haha